OBAKitCore/Location/Regions/RegionsService.swift

```diff
@@ -204,17 +207,10 @@ public class RegionsService: NSObject, LocationServiceDelegate {
 
     // MARK: - Custom Regions
     /// Adds the provided custom region to the RegionsService.
-    /// If an existing custom region with the same `regionIdentifier` exists, the new region replaces the existing region.
```

Comment:
With this new file manager service, what happens if an existing custom region with the same `regionIdentifier` exists?